assets/aws/files/make-amis-public.sh

```diff
@@ -6,7 +6,7 @@ REGION_LIST="us-east-1 us-east-2 us-west-1 us-west-2 ap-south-1 ap-northeast-2 a
 
 # Exit if oss/ent parameters not provided
 if [[ "$1" == "" ]]; then
-    echo "Usage: $(basename $0) [oss/ent/ent-fips]"
+    echo "Usage: $(basename "$0") [oss/ent/ent-fips]"
```

Comment:
Argh, think I missed this one.
```suggestion
    # shellcheck disable=SC2086
    echo "Usage: $(basename $0) [oss/ent/ent-fips]"
```
(not sure if it's SC2086 or SC2046)